packages/process/_process.pony

```diff
@@ -34,13 +34,95 @@ primitive _EAGAIN
 primitive _EINVAL
   fun apply(): I32 => 22
 
+primitive WNOHANG
```

Comment:
Maybe a Comment here to describe what `WNOHANG` represents. Also, would it be possible to make it `_WNOHANG`?

Reply:
The _ is a good catch. That should be changed.